src/components/ContentSidebar/ContentSidebar.js

```diff
@@ -628,7 +645,13 @@ class ContentSidebar extends PureComponent<Props, State> {
      */
     fetchFile(id: string, forceFetch: boolean = false): void {
         if (SidebarUtils.canHaveSidebar(this.props)) {
-            this.api.getFileAPI().file(id, this.fetchFileSuccessCallback, this.errorCallback, forceFetch, true);
+            this.api
+                .getFileAPI()
+                .file(id, this.fetchFileSuccessCallback, this.fetchFileErrorCallback, forceFetch, true);
+
+            this.setState({
+                isFileLoading: true
```

Comment:
Should this be a generic? isBusy or isLoading?

Reply:
I named it ```isFileLoading``` to make it more clear about what is actually loading/busy. I can change it if you feel strongly that it should be changed, but I think I prefer it as is